src/lib.rs

```diff
@@ -53,7 +45,7 @@ impl InOutFuncs for ulid {
 
 impl IntoDatum for ulid {
     #[inline]
-    fn into_datum(self) -> Option<Datum> {
+    fn into_datum(self) -> Option<pg_sys::Datum> {
```

Comment:
Please import `pg_sys::Datum as SysDatum` and use it. I like to always have imports.